audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,16 +113,50 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The audio worklet node is used by default.
+    this.useAudioWorkletCheckBox_ = document.createElement('input');
+    this.useAudioWorkletCheckBox_.type = 'checkbox';
+    this.useAudioWorkletCheckBox_.checked = true;
+    const audioworkletLabel = document.createElement('label');
+    audioworkletLabel.textContent =
+        'Use Audio Worklet (if unchecked, use Script Processor Bitcrusher).';
+
+    this.useAudioWorkletCheckBox_.addEventListener(
+        'change', this.switchBitcrusherNode.bind(this));
+
+    let container = document.getElementById(containerId);
+    container.appendChild(this.useAudioWorkletCheckBox_);
+    container.appendChild(audioworkletLabel);
+  }
+
+  /**
+   * Switch between whether the audio worklet node or script processor node
+   * connects to the master gain node.
+   */
+  switchBitcrusherNode() {
+    if (this.useAudioWorkletCheckBox_.checked) {
+       this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
```

Comment:
Does it really make sense in the demo to be able to dynamically switch between the script processor and the worklet?  At the very least the delays are different so it will sound funny.

I was kind of expecting the user to pick one mode and not dynamically switching while the music is playing.